pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -1054,6 +1054,7 @@ func handleCreate(ctx *domainContext, key string, config *types.DomainConfig) {
 		VncDisplay:         config.VncDisplay,
 		VncPasswd:          config.VncPasswd,
 		State:              types.INSTALLED,
+		VmConfig:           config.VmConfig,
```

Comment:
seems to be unused

Reply:
It is used in the form of vCpus; it is a #include of a struct in another struct so all of the field names show up in the parent struct.